roles/system_settings/tasks/dns.yml

```diff
@@ -0,0 +1,54 @@
+- name: Append the DNS servers
```

Comment:
maybe we need to split this task into 3 blocks:  DNS servers, DNS domains, and DNS hostname, what do you think?